src/main/java/by/exadel/internship/config/WebConfig.java

```diff
@@ -1,16 +1,21 @@
 package by.exadel.internship.config;
 
+import by.exadel.internship.interceptor.LoggerInterceptor;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @Configuration
 public class WebConfig implements WebMvcConfigurer {
 
     @Override
     public void addCorsMappings(CorsRegistry registry) {
-
         registry.addMapping("/**");
+    }
 
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        registry.addInterceptor(new LoggerInterceptor());
```

Comment:
I believe you should delegate interceptor creation and dependency injection to Spring.

Reply:
Fixed